resources/js/menus/LeftMenu.vue

```diff
@@ -48,6 +48,52 @@
 			</template>
 		</Menu>
 		<AboutLychee v-model:visible="openLycheeAbout" />
+		<div class="mt-auto">
+			<Menu :model="profileItems" v-if="initData" class="!border-none">
+				<template #item="{ item, props }">
+					<template v-if="item.access">
+						<router-link v-if="item.route" v-slot="{ href, navigate }" :to="item.route" custom>
+							<a v-ripple :href="href" v-bind="props.action" @click="navigate">
+								<MiniIcon :icon="item.icon ?? ''" :class="'w-3 h-3'" />
+								<span class="ml-2">
+									<!-- @vue-ignore -->
+									{{ $t(item.label) }}
+								</span>
+								<SETag v-if="item.seTag" />
+							</a>
+						</router-link>
+						<a v-if="item.url" v-ripple :href="item.url" :target="item.target" v-bind="props.action">
+							<MiniIcon :icon="item.icon ?? ''" :class="'w-3 h-3'" />
+							<span class="ml-2">
+								<!-- @vue-ignore -->
+								{{ $t(item.label) }}
+							</span>
+							<SETag v-if="item.seTag" />
+						</a>
+						<a v-if="!item.route && !item.url" v-ripple v-bind="props.action">
```

Comment:
if it a `<a` tag without a route or url?

Reply:
Yes, it is using the v-bind to execute the command.